plugins/module_utils/network/dcnm/dcnm.py

```diff
@@ -262,6 +276,47 @@ def dcnm_get_ip_addr_info(module, sw_elem, ip_sn, hn_sn):
 #             raise Exception(json.dumps(msg_dict))
 
 
+def get_fabric_details(module, fabric):
```

Comment:
WE already have a function to get FABRIC details. Is this something different? Otherwise use the existing one

Reply:
The function which is there already gets the fabric inventory details. Here we get the fabric details